src/define_component.ts

```diff
@@ -0,0 +1,99 @@
+import { Wunderbaum } from "./wunderbaum";
+
+function appendCssLink(url: string, shadowRoot?: ShadowRoot) {
+  let link = document.createElement("link");
+  link.setAttribute("rel", "stylesheet");
+  link.setAttribute("href", url);
+  if (shadowRoot) {
+    shadowRoot.appendChild(link);
```

Comment:
Wouldn't it be better to include the style via css import and assign to components adoptet stylesheets?